js/index.js

```diff
@@ -135,21 +182,14 @@ var LocalNotify = Class(Emitter, function (supr) {
 		}
 		var time = date.getTime() / 1000;
 		if (opts.delay) {
-			var delay = opts.delay;
-			if (delay.seconds) {
-				time += delay.seconds;
-			}
-			if (delay.minutes) {
-				time += delay.minutes * 60;
-			}
-			if (delay.hours) {
-				time += delay.hours * 60 * 60;
-			}
-			if (delay.days) {
-				time += delay.days * 60 * 60 * 24;
-			}
+			opts.utc = time + this.calculateTime(opts.delay);
+		}
+
+		if(opts.repeat) {
+			var time  = this.calculateTime(opts.repeat);
```

Comment:
variable have either function or global scope. There is no block scope. No point in declaring a variable here